da/da.go

```diff
@@ -14,6 +15,14 @@ import (
 	pb "github.com/rollkit/rollkit/types/pb/rollkit"
 )
 
+const (
+	// submitTimeout is the timeout for block submission
+	submitTimeout = 60 * time.Second
```

Comment:
Is this something we're fine with hardcoding regardless of the DA layer? Do you think we should make this configurable for the future? Thoughts? @gupadhyaya

Reply:
@tuxcanfly 
* any reason for choosing 60s not 30s? 
* do you need two separate timeouts for submit and retrieve? 
* @Manav-Aggarwal i think we can use set defaults for now and later make it configurable when we have multiple DA support. this is still an safe upper-bound.

Reply:
> any reason for choosing 60s not 30s?

in extreme cases e.g. inscriptions flood, the core endpoint might timeout accepting transactions, it's a sync method right now (`BroadcastTx`).

> do you need two separate timeouts for submit and retrieve?

Yes as both have different contexts and implementations (e.g. light nodes might require more time than bridge nodes).

> Is this something we're fine with hardcoding regardless of the DA layer? 

agreed, defaults for now but should be configurable.